homeassistant/components/rainmachine/__init__.py

```diff
@@ -79,9 +80,20 @@
 CONF_SOLARRAD = "solarrad"
 CONF_TEMPERATURE = "temperature"
 CONF_TIMESTAMP = "timestamp"
+CONF_UNITS = "units"
+CONF_VALUE = "value"
 CONF_WEATHER = "weather"
 CONF_WIND = "wind"
 
+# Config Validator for Flow Meter Data
+CV_FLOW_METER_VALID_UNITS = {
+    "clicks",
```

Comment:
Just realized: let's use Home Assistant constants where we can.

https://github.com/home-assistant/core/blob/c5688072fdb9ce40eea55648fb3b95c043433030/homeassistant/const.py#L561

https://github.com/home-assistant/core/blob/c5688072fdb9ce40eea55648fb3b95c043433030/homeassistant/const.py#L559

https://github.com/home-assistant/core/blob/c5688072fdb9ce40eea55648fb3b95c043433030/homeassistant/const.py#L564

...and create one for `clicks`.

Reply:
I was going to use these, but they don't match what Rainmachine is expecting (e.g. `litre` vs `L`). So either we use the RainMachine versions or we add another step to convert them before calling from the controller.

What's your preference?

Reply:
Ah, good point. I don't think a conversion step provides real value, so okay to ignore and go with what you have. 👍🏻